e2e/cypress/support/saml_commands.js

```diff
@@ -81,33 +32,6 @@ Cypress.Commands.add('doSamlLogout', (settings = {}) => {
     });
 });
 
-Cypress.Commands.add('doSamlLogoutFromSignUp', () => {
-    cy.checkSignUpPageSection();
-    cy.findByText('Logout').should('be.visible').click();
-});
-
-Cypress.Commands.add('skipOrCreateTeam', (settings, userId) => {
-    let teamName = '';
-
-    cy.get('body').then((body) => {
-        if (body.text().includes('Create a team')) {
-            cy.checkCreateTeamPage(settings);
-
-            teamName = 't' + userId.substring(0, 14);
-            cy.findByText('Create a team').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
-                cy.get('input[id="teamNameInput"]').should('be.visible').type(teamName, {force: true}).wait(TIMEOUTS.TINY).then(() => {
-                    cy.findByText('Next').should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
-                        cy.findByText('Finish').should('be.visible').click().wait(TIMEOUTS.TINY).then(() => {
-                            settings.teamName = teamName;
-                        });
-                    });
-                });
-            });
-        }
-        settings.teamName = teamName;
-    });
-});
-
 Cypress.Commands.add('getInvitePeopleLink', (settings = {}) => {
     cy.checklLeftSideBar(settings);
```

Comment:
This was a previous typo error but since corrected at and moved to [ldap_commands](https://github.com/mattermost/mattermost-webapp/pull/5114/files#diff-6ce6790c7180b3cd18b6654223507b44R31), needs to update as well. Same case on [L25](https://github.com/mattermost/mattermost-webapp/pull/5114/files#diff-aa9dc118678b8174561f7ad3d9aa1881L74).
```suggestion
    cy.checkLeftSideBar(settings);
```

Reply:
Nice catch....